src/pages/public_registration/preview_file/PreviewImg.tsx

```diff
@@ -1,10 +1,18 @@
 import { Box } from '@mui/material';
+import { dataTestId } from '../../../utils/dataTestIds';
 import { CommonPreviewProps } from './PreviewFile';
 
 interface PreviewImgProps extends CommonPreviewProps {
   imgAlt: string;
 }
 
 export const PreviewImg = ({ url, imgAlt, ...props }: PreviewImgProps) => (
-  <Box component="img" sx={{ maxWidth: '100%', maxHeight: '25rem' }} src={url} alt={imgAlt} {...props} />
+  <Box
+    data-testid={dataTestId.registrationLandingPage.filePreview}
```

Comment:
data-testid vil være her om det er `alt` som vises også, så det er kanskje noe å være klar over. bør sikkert sjekke for bestemte HTML tags i tillegg